testsuite/tests/libratest/smoke_test.rs

```diff
@@ -116,20 +117,29 @@ impl TestEnvironment {
             false,
             /* faucet server */ None,
             Some(mnemonic_file_path),
+            waypoint,
         )
         .unwrap()
     }
 
-    fn get_validator_ac_client(&self, node_index: usize) -> ClientProxy {
+    fn get_validator_ac_client(
+        &self,
+        node_index: usize,
+        waypoint: Option<Waypoint>,
+    ) -> ClientProxy {
         let port = self.validator_swarm.get_ac_port(node_index);
-        self.get_ac_client(port)
+        self.get_ac_client(port, waypoint)
```

Comment:
we probably can generate the genesis waypoint in LibraSwarm and fill in for all clients